src/ixdat/calculators/ms_calculators.py

```diff
@@ -0,0 +1,829 @@
+        new_cal_list = []
+        for cal in self.ms_cal_results:
+            cal = MSCalResult(
+                name=cal.name,
+                mass=cal.mass,
+                mol=cal.mol,
+                F=cal.F * scale_factor,
+                cal_type=cal.cal_type + " scaled",
+            )
+            new_cal_list.append(cal)
+        calibration_as_dict["ms_cal_results"] = new_cal_list
+        del calibration_as_dict["ms_cal_result_ids"]
+        # ^ FIXME: ms_cal_result_ids via MemoryBackend
+        calibration_as_dict["name"] = calibration_as_dict["name"] + " scaled"
+        return self.__class__.from_dict(calibration_as_dict)
+
+    @classmethod
+    def read(cls, path_to_file):
+        """Read an MSCalibration from a json-formatted text file"""
+        with open(path_to_file) as f:
+            obj_as_dict = json.load(f)
+        # put the MSCalResults (exported as dicts) into objects:
+        obj_as_dict["ms_cal_results"] = [
+            MSCalResult.from_dict(ms_cal_as_dict)
+            for ms_cal_as_dict in obj_as_dict["ms_cal_results"]
+        ]
+        return cls.from_dict(obj_as_dict)
+
+    def export(self, path_to_file=None):
+        """Export an ECMSCalibration as a json-formatted text file"""
+        path_to_file = path_to_file or (self.name + ".ix")
+        self_as_dict = self.as_dict()
+        # replace the ms_cal_result ids with the dictionaries of the results themselves:
+        del self_as_dict["ms_cal_result_ids"]
+        self_as_dict["ms_cal_results"] = [cal.as_dict() for cal in self.ms_cal_results]
+        with open(path_to_file, "w") as f:
+            json.dump(self_as_dict, f, indent=4)
+
+    @classmethod
+    def from_siq(cls, siq_calculator):
+
+        # A complication is that it can be either a Calibration or a SensitivityList.
+        # Either way, the sensitivity factors are in `sf_list`:
+        ms_cal_results = [MSCalResult.from_siq(cal) for cal in siq_calculator.sf_list]
+        # if it's a Calibration, we want the metadata:
+        try:
+            calibration = cls(
+                name=siq_calculator.name,
+                date=siq_calculator.date,
+                setup=siq_calculator.setup,
+                ms_cal_results=ms_cal_results,
+            )
+        # if not, we just want the data:
+        except AttributeError:
+            calibration = cls(ms_cal_results=ms_cal_results)
+        return calibration
+
+    def to_siq(self):
+        if not plugins.use_siq:
+            raise QuantificationError(
+                "`MSCalibration.to_siq` only works when using "
+                "`spectro_inlets_quantification` "
+                "(`ixdat.plugins.activate_siq()`). "
+            )
+        cal_list = [cal.to_siq() for cal in self.ms_cal_results]
+        return plugins.siq.Calculator(
+            name=self.name,
+            date=self.date,
+            setup=self.setup,
+            cal_list=cal_list,
+        )
+
+
+class MSInlet:
```

Comment:
Why is the MSInlet in the calculators module? shouldnt this still be in the corresponding technique module?

Reply:
I think it belongs here because it has more to do with the act of calculating than the act of measuring.